docs/index.html

```diff
@@ -22,6 +22,9 @@ <h4>Default</h4>
       <div id="viewer-default" class="viewer-3d"></div>
       <h4>Without Popup</h4>
       <div id="viewer-no-popup" class="viewer-3d"></div>
+      <h4>With change SKU</h4>
+      <button type="button" class="btn btn-primary" id="button-changeSku">SKU 1 </button>
+      <div id="viewer-changeSku" class="viewer-3d"></div>
```

Comment:
Best practices de ID HTML: `Keep it all lower-case and use underscores or hyphens`

Reply:
legal @FelipeR2U acho essa é uma boa
de onde vc pegou essa referência? Pode adicionar no [notion](https://www.notion.so/r2u/Dev-fc3a1f1eaab14bc8a4157f08cdfeba49)? na parte de best practices

Reply:
https://courses.cs.washington.edu/courses/cse154/17au/styleguide/html-css/naming-conventions-html.html

O que eu copiei peguei de um stack Overflow, mas tinha visto primeiro [aqui](https://courses.cs.washington.edu/courses/cse154/17au/styleguide/html-css/naming-conventions-html.html)

Reply:
kebab-case ?

Reply:
Não manjava dessa 😆